.development/googlesheets_dataclean.R

```diff
@@ -0,0 +1,40 @@
+library(tidyverse)
+library(googlesheets4)
```

Comment:
Could you do `renv::snapshot()` to record the packages in the *renv.lock* file? Also can you add them to the *DESCRIPTION* file as well?